server/catgenome/src/main/java/com/epam/catgenome/manager/project/ProjectManager.java

```diff
@@ -388,7 +394,38 @@ public Project create(final Project project, Long parentId) {
                 dataItems.forEach(i -> fillFileTypeLists(i, newGeneFiles, newVcfFiles));
             }
         }
-
+        if (!newProject && !CollectionUtils.isEmpty(loadedProject.getNotes())) {
```

Comment:
Could you please extract this logic into a separate method? Original method is already too long